HTML:CSS learning project/index.html

```diff
@@ -6,23 +6,25 @@
   <title>Road to web developer</title>
   <link rel="stylesheet" href="style.css">
 </head>
-<body>
-  <header>
+<body class="font">
```

Comment:
There is an exception to the body. If you know that styles should be applied globally and remain the same, then you could use a global selector for the body. 

For example the font. Let's assume, that it'll never change. In that case, setting styles on global elements is reasonable and exceptional.